frontend/app/src/views/settings/Settings.vue

```diff
@@ -36,7 +36,7 @@ export default class Settings extends Vue {
       routeTo: '/settings/accounting'
     },
     {
-      name: 'User & Security',
+      name: 'Data & Security',
       routeTo: '/settings/user-security'
     },
```

Comment:
Need to update the route here from `settings/user-security` to `settings/data-security`